packages/orbit-components/src/tmp-tailwind/Switch/index.tsx

```diff
@@ -0,0 +1,59 @@
+"use client";
+
+import * as React from "react";
+import cx from "clsx";
+
+import Circle from "../../icons/Circle";
+import handleKeyDown from "../../utils/handleKeyDown";
+import type { Props } from "../../Switch/types";
+
+const Switch = React.forwardRef<HTMLInputElement, Props>(
+  ({ onChange, checked, dataTest, id, icon, onBlur, onFocus, disabled, ariaLabelledby }, ref) => {
+    return (
+      <div className="inline-block">
```

Comment:
Isn't this a label? Probably doesn't matter as we're overriding the `display` attr.